pkg/scripts/os_flatcar.go

```diff
@@ -251,6 +251,8 @@ func criToolsVersion(cluster *kubeoneapi.KubeOneCluster) string {
 		return "1.30.1"
 	case 31:
 		return "1.31.1"
+	case 32:
+		return "1.31.1"
```

Comment:
This should be [1.32.0](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.32.0).

Reply:
:heavy_check_mark: